stream/sentinel/src/sentinel.py

```diff
@@ -235,13 +240,15 @@ def _create_observable(self, data):
         else:
             passive_only = "false"
 
-        self.helper.log_info("[CREATE] Processing data {" + internal_id + "}")
+        self.helper.log_info(
```

Comment:
This method to display logs is deprecated now, you can use:

```suggestion
        self.helper.connector_logger.info(
```